registration/tests/enhanced_backend.py

```diff
@@ -0,0 +1,258 @@
+import datetime
```

Comment:
Actually it is not all copied as tests for the default backend use `RegistrationProfile` instead of `SupervisedRegistrationProfile`. I could alter the original `DefaultBackendViewTests` class to hold the registration profile it will use as a class variable and override it when subclassing, but I am not sure you would want that. If you think it would be better let me know.